### ISSUE TITLE
site definition

### ISSUE THREAD

**Issue by epaulson** [BrickSchema/Brick#369]
A long time back we had a discussion about how to use 'site' as a grouping. In this commit:
https://github.com/BrickSchema/Brick/commit/f7f885ad30e08837f39c072df379e6097616342c

we updated the definition of site to say that ```brick:site``` is "Typically used as the encapsulating location for a collection of Brick entities through the hasSite/isSiteOf relationship"

I thought we decided against adding a 'hasSite/isSiteOf' relationship, but at any rate we haven't added that relationship, so we should either add the relationships or remove this clause of the definition. 

(I think 'hasPart' from site to building is good enough and we don't need a 'hasSite' relationship. But I know Haystack uses it and I can live with 'hasSite' if it makes life easier)

**Comment by jbkoh** [comment]
I recall too that we decided `hasSite` won't be necessary in one of our meetings. The beginning of the discussion is [here](https://github.com/BrickSchema/Brick/pull/180#issuecomment-710439929), though I think we didn't write down the decision.

The domain of `hasSite` is unclear and could be used widely differently. I still vote for sticking to `hasPart`.

**Comment by gtfierro** [comment]
Yes, I remember this discussion! Definition will be fixed in #383
